test/operators-simple.js

```diff
@@ -269,31 +273,31 @@ QUnit.test("Goal  'r(X,Y).'", function(assert) {
   session.query("r(X,Y).");
 
   session.answer(function(success) {
-	console.log(success);
+    // console.log(success);
     assert.equal(success.links.X.id, "a", "X is a");
     assert.equal(success.links.Y.id, ",", "Y is a couple (',')");
     assert.equal(success.links.Y.args[0].id, "b", "First element in couple is b");
     assert.equal(success.links.Y.args[1].id, "c", "Second element in couple is c");
   });
 
   session.answer(function(success) {
-	console.log(success);
+    // console.log(success);
     assert.equal(success.links.X.id, ",", "X is a couple (',')");
     assert.equal(success.links.X.args[0].id, "c", "First element in couple is c");
     assert.equal(success.links.X.args[1].id, "d", "Second element in couple is d");
     assert.equal(success.links.Y.id, "e", "Y is e");
   });
   
    session.answer(function(success) {
-	console.log(success);
+    // console.log(success);
     assert.equal(success.links.X.id, "i", "X is i");
     assert.equal(success.links.Y.id, ",", "Y is a couple (',')");
     assert.equal(success.links.Y.args[0].id, "j", "First element in couple is j");
     assert.equal(success.links.Y.args[1].id, "k", "Second element in couple is k");
   });
   
    session.answer(function(success) {
-	console.log(success);
+    // console.log(success);
```

Comment:
I commented out all the `console.log`'s in the tests because they made the QUnit terminal output unreadable since they output large objects.